graphql/database/users/__tests__/getShouldShowYahooPrompt.test.js

```diff
@@ -14,12 +18,19 @@ describe('shouldShowYahooPrompt tests', () => {
     expect.assertions(1)
     const getShouldShowYahooPrompt = require('../getShouldShowYahooPrompt')
       .default
-    getUserFeature.mockResolvedValueOnce(
-      new Feature({
-        featureName: YAHOO_SEARCH_EXISTING_USERS,
-        variation: false,
-      })
-    )
+    getUserFeature
+      .mockResolvedValueOnce(
+        new Feature({
+          featureName: YAHOO_SEARCH_EXISTING_USERS,
+          variation: false,
+        })
+      )
+      .mockResolvedValueOnce(
+        new Feature({
+          featureName: YAHOO_SEARCH_NEW_USERS,
+          variation: 'Google',
+        })
+      )
```

Comment:
This test says it's testing when the user is **in** the experiment, so should one of these features be the experiment variation?

Reply:
Also, is this test intending to do the same thing as the one below?